src/channels/router_mqtt_channel.erl

```diff
@@ -429,20 +429,20 @@ connect(URI, DeviceID, Name) ->
     case uri_string:parse(URI) of
         #{
             host := Host,
-            port := Port,
             scheme := Scheme
         } = Map when
             Scheme == <<"mqtt">> orelse
                 Scheme == <<"mqtts">>
         ->
+            Port = maps:get(port, Map, 1883),
```

Comment:
Should we default to `8883` if the scheme is `mqtts`?

Reply:
Good idea!